### ISSUE TITLE
chuck.exe crashing when sending an OscOut message

### ISSUE THREAD

**Issue by MichaPau** [ccrma/chuck#98]
Trying to send a simple OscOut message crashes chuck.exe.

```
OscOut oout;

oout.dest("localhost", 6449);

oout.start("/playNote");
oout.add(60);
oout.add(0.5);
oout.add("mandoline");

oout.send();
```

This simple example code (from the Kadenze course: Introduction to Programming for Musicians and Digital Artists) crashes on **Windows7**. Run from miniAudicle or from the command line.

The class OscOut is not recognized as such in miniAudicle. It doesn't change the colour to red/purple like other classes do. It stays black like normal text. (see the screen-shot)
![audicle_oscout](https://user-images.githubusercontent.com/4903900/32415317-5bedd70a-c238-11e7-9b50-523d0ede8db8.png)
)  

I am not the only one having this issue. 
[see the forum entry on Kadenze](https://www.kadenze.com/forums/introduction-to-programming-for-musicians-and-digital-artists-sessions-session-8-live-control-keyboard-mouse-midi-osc/threads/oscout-oscin)

**Comment by lathertonj** [comment]
Please see the comments of #93 for the older, more stable API for OSC messages. Try using  OscSend, OscRecv, OscEvent instead of OscIn, OscOut, OscMsg.

**Comment by gewang** [comment]
sorry this has taken literally years; 2023 has been a time of chuck renewal... This issue at last should be resolved (commit 3107379). The issue was that on Windows, the network socket was being closed incorrectly in the version of liblo we are using. This fix will be part of the next release (chuck-1.5.1.3).

Please feel free to re-open if issue persist. Also, you are invited to join us on the [ChucK Community Discord server](https://discord.gg/ENr3nurrx8)! Thanks all for attending to this once upon a time!
